AddOnInfo.plist.in

```diff
@@ -1,34 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!-- https://developer.apple.com/documentation/bundleresources/information-property-list -->
 <plist version="1.0">
 	<dict>
-		<key>CFBundleDevelopmentRegion</key>
-		<string>English</string>
-		<key>CFBundleExecutable</key>
-		<string>@MACOSX_BUNDLE_EXECUTABLE_NAME@</string>
-		<key>CFBundleGetInfoString</key>
-		<string>@MACOSX_BUNDLE_INFO_STRING@</string>
-		<key>CFBundleIconFile</key>
-		<string>ArchiCADPlugin</string>
-		<key>CFBundleIdentifier</key>
-		<string>@MACOSX_BUNDLE_GUI_IDENTIFIER@</string>
 		<key>CFBundleInfoDictionaryVersion</key>
 		<string>6.0</string>
-		<key>CFBundleLongVersionString</key>
-		<string>@MACOSX_BUNDLE_LONG_VERSION_STRING@</string>
-		<key>CFBundleName</key>
-		<string>@MACOSX_BUNDLE_BUNDLE_NAME@</string>
 		<key>CFBundlePackageType</key>
 		<string>.APX</string>
-		<key>CFBundleSignature</key>
-		<string>GSAP</string>
```

Comment:
I have removed `CFBundleSignature`, `CFBundleLongVersionString` and `CFBundleGetInfoString` as these are obsolete and unused.

Reply:
How do these match the XCode settings? If the developer creates an Xcode project and builds from there (e.g., because of code signing), do these behave "well"?